vignettes/appeals.Rmd

```diff
@@ -50,6 +50,14 @@ library(sf)
 ptaxsim_db_conn <- DBI::dbConnect(RSQLite::SQLite(), here("./ptaxsim.db"))
 ```
 
+```{r, echo=FALSE}
+# This is needed to build the vignette using GitHub Actions
+ptaxsim_db_conn <- DBI::dbConnect(
+  RSQLite::SQLite(),
+  Sys.getenv("PTAXSIM_DB_PATH")
+)
```

Comment:
**[Question, non-blocking]** What's the relationship between this `DBI::dbConnect()` call and the one that precedes it on line 50? Is the idea just that this will overwrite the preceding `ptaxsim_db_conn` definition on CI, but locally we can skip this call and use the one on line 50 instead?

Reply:
You got it. This call is hidden in the actual output vignette HTML just because I think it's simpler to show a file path instead of an env var setup, but the env var is necessary for it to work on CI.